content/02.main-text.md

```diff
@@ -21,6 +21,34 @@ Although we requested that some authors participate for their specific expertise
 
 
+### Collaborative writing and reviewing
+
+There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats, to LaTeX-based systems for more technical writing [@url:https://www.overleaf.com/ @url:https://www.authorea.com/].
+`TODO: survey other existing options, see if there are better references`
+These platforms typically offer version control and other features to support multi-author editing.
+`TODO: some use git in the back end, is that relevant?`
+However, their critical limitation for open writing is that they give all authors permission to modify all parts of the manuscript.
+This makes it difficult to maintain consistent content and tone and is susceptible to malicious edits.
+`TODO: are wikis, especially Wikipedia, relevant as an alternative approach?`
+
+We adopted standard software development strategies in order to enable any contributor to edit any part of the manuscript but enforce discussion and review of all proposed changes.
+The GitHub platform provided support for organizing and editing the manuscript.
+We used GitHub _issues_ for organization, opening a new issue for each paper under consideration.
+Within the issue, contributors summarized the research, discussed it (sometimes with the original authors), and assessed its relevance to the review.
+Issues also served as an open to-do list and a forum for debating the main message, themes, and topics of the review.
+
+GitHub and the underlying git version control system also structured the writing process.
+An official master version could be _forked_ by individual contributors.
+A contributor could then add and revise any files, grouping these changes into _commits_.
+When the changes were ready to be reviewed, the series of commits could be submitted as a _pull request_ through GitHub, which notified other authors of the pending changes.
+GitHub's review interface allows anyone to comment on the changes, globally or at specific lines, asking questions or requesting modifications from the author.
+Once all requested modifications were made, the manuscript owners, a subset of authors with elevated GitHub permissions, could formally approve the pull request and merge the changes into the official master version.
+`TODO: need a figure with a flowchart showing this process`
+`TODO: could cite https://medium.com/towards-data-science/opportunities-and-obstacles-for-deep-learning-in-biology-and-medicine-6ec914fe18c2, the figure is nice`
+The entire process can be orchestrated through GitHub with a web browser if a contributor does not want to use a git client on their own computer.
+We found that this workflow was the best compromise between fully unrestricted editing and a more heavily-structured approach that limits the authors or the sections they can edit.
+In addition, authors are associated with their commits, making it easy for contributors to receive credit for their work.
```

Comment:
Perhaps we could add some citations about credit problems in academia or ghostwriting, such as https://doi.org/10.1371/journal.pmed.1000023.